go.mod

```diff
@@ -45,17 +45,17 @@ require (
 replace (
 	github.com/niemeyer/pretty => github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e
 	go.etcd.io/etcd => go.etcd.io/etcd v0.0.0-20200410171415-59f5fb25a533
-	google.golang.org/grpc => google.golang.org/grpc v1.27.0
```

Comment:
I believe can drop the `replace` statement for `google.glang.org/grpc` since it matches the `requires` version now.

Reply:
Fixed